streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -1201,6 +1201,9 @@ private void verifyMaxInFlightRequestPerConnection(final Object maxInFlightReque
         // disable auto topic creation
         consumerProps.put(ConsumerConfig.ALLOW_AUTO_CREATE_TOPICS_CONFIG, "false");
 
+        // enable early return on metadata
+        consumerProps.put(ConsumerConfig.LONG_POLL_MODE_CONFIG, ConsumerConfig.LONG_POLL_RETURN_ON_RECORDS);
```

Comment:
It should be `LONG_POLL_RETURN_ON_RESPONSE` rather than `LONG_POLL_RETURN_ON_RECORDS`

Reply:
Oy, you are right. Thanks!